benchmarks/lucet-benchmarks/src/seq.rs

```diff
@@ -240,7 +240,8 @@ fn run_fib<R: RegionCreate + 'static>(c: &mut Criterion) {
 /// Run a trivial WASI program.
 fn run_hello<R: RegionCreate + 'static>(c: &mut Criterion) {
     fn body(inst: &mut InstanceHandle) {
-        inst.run("_start", &[]).unwrap();
+        const START: &'static str = "_start";
```

Comment:
`"_start"` _is_ a `&'static str` by definition, as all string literals are, so you don't have to actually store it anywhere before using - it's entirely valid and correct to write `inst.run("_start", &[])`, here, and similarly for other places that take `&str`

Reply:
I thought I had tried that but as a newb perhaps I made a mistake.  I was getting compiler errors complaining about lifetimes of the &str which was why I made explicit static.  Regardless I will to do as you request. Thanks

Reply:
I see from above you were a bit tongue and cheek for this change but its done anyway.  Looks better too :-)